handlers.go

```diff
@@ -132,6 +132,14 @@ func isUserPath(username, path string) bool {
 	return len(path) == len(prefix) || path[len(prefix)] == '/'
 }
 
+func isMetricsPath(path string) bool {
+	prefix := "/metrics"
+	if !strings.HasPrefix(path, prefix) {
+		return false
+	}
+	return len(path) == len(prefix) || path[len(prefix)] == '/'
```

Comment:
What does this match? `/metrics` and `/metrics/`?

I'd like to standardize on `/metrics` (the literal string, without the slash), so the test is so much simpler: `return path == "/metrics"`. I think we don't need the flexibility here, it just complicates the code.